src/mscorlib/ref/mscorlib.cs

```diff
@@ -15,6 +15,12 @@ public static partial class EnvironmentAugments
         public static void FailFast(string message, System.Exception error) { }
         public static string[] GetCommandLineArgs() { throw null; }
     }
+
+    public static partial class AppDomainAugments
```

Comment:
It is case by case - GetAssemblies() should go to AssemblyLoadContext; UnhandledException should go to EnvironmentAugments, I think.